package.json

```diff
@@ -31,6 +30,7 @@
     "@angular/cli": "~8.2.2",
     "@angular/compiler-cli": "~8.2.0",
     "@angular/language-service": "~8.2.0",
+    "@fortawesome/free-solid-svg-icons": "^5.15.4",
```

Comment:
also this seemed to have been missing so I added it back... I hope it is there for the next reviewer! If not, let me know so I can do some more troubleshooting with it

Reply:
It was missing for me too - but I think Laura has it fixed up now. :)